### PR TITLE
WRN-3498: Migrated unit tests for Button to testing-library

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Added unit tests for Button

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-3498

### Comments

Enact-DCO-1.0-Signed-off-by: Daniel Stoian daniel.stoian@lgepartner.com

### CODE REVIEW

**Review by stanca-pop-lgp — COMMENTED** [review]
Just one suggestion, to add quotes in test descriptions for attributes, classes and their values (like for the first test, for example, or somehow that it is easier to read).

File: Button/tests/Button-specs.js [hunk 1]
```diff
@@ -1,58 +1,169 @@
...
 
 describe('Button', () => {
+	test('should have \'disabled\' HTML attribute when \'disabled\' prop is provided', () => {
+		render(<Button disabled>I am a disabled Button</Button>);
+		const button = screen.getByRole('button');
+
+		expect(button).toHaveAttribute('disabled');
+	});
+
+	test('should have default minWidth', function () {
```
Comment (alexandrumorariu) [inline Button/tests/Button-specs.js]: We should replace function () with arrow functions.

**Review by alexandrumorariu — APPROVED** [review]
LGTM

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

**Pull request merged**